bagel/pheno_utils.py

```diff
@@ -352,9 +366,21 @@ def validate_data_dict(data_dict: dict) -> None:
     try:
         jsonschema.validate(data_dict, DICTIONARY_SCHEMA)
     except jsonschema.ValidationError as e:
+        # TODO: When *every* item in an input JSON is not schema valid,
+        # jsonschema.validate will raise a ValidationError for only *one* item among them.
+        # Weirdly, the item that is chosen can vary, possibly due to jsonschema internally processing/validating
+        # items in an inconsistent order.
+        # You can reproduce this by running `bagel pheno` on the example_invalid input pair.
+        # This isn't necessarily a problem as the error will not be wrong, but if we care about
+        # returning ALL invalid items, we may want to use something like a Draft7Validator instance instead.
```

Comment:
Very interesting. I wasn't aware of the difference between `jsonschema.validate` and `jsonschema.Validator.validate`. would you mind opening an issue for potentially refactoring to using the second form in the future?

Reply:
Done! See https://github.com/neurobagel/bagel-cli/issues/338